src/js/views/awardsRecords.js

```diff
@@ -120,7 +105,7 @@ function getPlayerAwards(p, awardType) {
     const formatYear = year => {
         return Object.keys(year).map(k => {
             const years = helpers.yearRanges(year[k].map(y => y.season)).join(', ');
-            return `${k} <small>(${years})</small>`;
+            return `${k} (${years})`;
```

Comment:
>in which case you can assemble it in AwardsRecords.js instead of here, similar to what you did with the player name link.

Ah ok it all makes sense now lol, hopefully will have this up before I have to start on real job this morning.